apps/src/lib/node/ledger/ethereum_node/oracle.rs

```diff
@@ -32,6 +37,12 @@ pub struct Oracle {
     /// A channel to signal that the ledger should shut down
     /// because the Oracle has stopped
     abort: Option<Sender<()>>,
+    /// How long the oracle should wait between checking blocks
+    backoff: Duration,
```

Comment:
This is crude linear backoff for now, mainly for the purpose of stopping the ledger from spamming the Ethereum RPC endpoint once it has caught up syncing, but there should be some more involved logic around this backoff later